Resources/public/js/profiling/GABackend.js

```diff
@@ -59,13 +59,17 @@ Ext.define('Modera.backend.googleanalytics.profiling.GABackend', {
         }
 
         if (target instanceof MF.activation.activities.AbstractActivity) {
-            ga('send', {
-                hitType: 'timing',
-                timingCategory: 'MJR',
-                timingVar: this.trackingPlugin.createToken(),
-                timingLabel: 'View activation',
-                timingValue: ms
-            });
+            if (typeof ga !== 'undefined') {
+                ga('send', {
+                    hitType: 'timing',
+                    timingCategory: 'MJR',
+                    timingVar: this.trackingPlugin.createToken(),
+                    timingLabel: 'View activation',
+                    timingValue: ms
+                });
+            } else {
+                console.error("Please, provide a GA token");
```

Comment:
Conventionally we include a FQCN where the reporting is done. So in you case it would look something like:

`
console.error("Modera.backend.googleanalytics.profiling.GABackend.onProfileComplete(): GA token is not configured, unable to send analytics.");
`